v4/weaviate/graphql/neartextbuilder.go

```diff
@@ -6,6 +6,68 @@ import (
 	"strings"
 )
 
+// fldMover is a type representing field names of a move sub query
+type fldMover string
+
+const (
+	fldMoverConcepts fldMover = "concepts"
+	fLDMoverForce    fldMover = "force"
+	fldMoverID       fldMover = "id"
+	fldMoverBeacon   fldMover = "beacon"
+	fldMoverObjects  fldMover = "objects"
+)
+
+// MoveParameters to fine tune Explore queries
+type MoveParameters struct {
+	// Concepts that should be used as base for the movement operation
+	Concepts []string
+	// Force to be applied in the movement operation
+	Force float32
+	// Movers _
+	Movers []MoverObject
+}
+
+func (m *MoveParameters) String() string {
+	concepts, _ := json.Marshal(m.Concepts)
```

Comment:
are we certain this can never error? not sure what kind of validation is performed prior to this point

Reply:
This should never returns an error.

To be certain I've added a new utility which marshal the slice above without returning any error